storybook/cypress/e2e/responsive-element.cy.ts

```diff
@@ -0,0 +1,28 @@
+describe("Responsive element", () => {
+  const responsiveElementSelector = "dso-responsive-element.hydrated";
+  beforeEach(() => {
+    cy.visit("http://localhost:45000/iframe.html?id=core-responsive-element--responsive-element");
+  });
+
+  it("should be accessible - default", () => {
+    cy.injectAxe();
+    cy.dsoCheckA11y(responsiveElementSelector);
+  });
+
+  it("should have size (small, medium, large) attribute depending on the viewport", () => {
+    cy.viewport(1312, 660);
+    cy.get(responsiveElementSelector).first().should("have.attr", "large");
+
+    cy.viewport(992, 660);
+    cy.get(responsiveElementSelector).first().should("have.attr", "medium");
+
+    cy.viewport(656, 660);
+    cy.get(responsiveElementSelector).first().should("have.attr", "large");
```

Comment:
Ik heb me nog nooit verdiept in dit component. Hoe werkt deze dan? `large` kan ook bestaan bij een smallere viewport?

Reply:
Klopt, responsive-element straalt de breedte van het element uit dat deze in kan nemen. Omdat ze in storybook in row > (responsive) cols zijn opgenomen krijgen ze verschillende breedtes op verschillende viewports.

Een large of medium zal niet bestaan op portrait mobiele viewport bijv. Dit zijn de viewport waardes waarop het component de breedte in woord "emit"

```
  {
    width: 624,
    alias: "large",
  },
  {
    width: 375,
    alias: "medium",
  },
  {
    width: 0,
    alias: "small",
  },
```